forgerock-authenticator/src/main/java/org/forgerock/android/auth/Account.java

```diff
@@ -38,21 +39,33 @@ public class Account extends ModelObject<Account> {
     private final String backgroundColor;
     /** Date this object was stored */
     private final Calendar timeAdded;
+    /** Authenticator Policies in a JSON String format */
+    private final String policies;
+    /** Name of the Policy locking the Account */
+    private String lockingPolicy;
+    /** Account lock flag */
+    private boolean lock;
     /** List of Mechanism objects associated with this account **/
     private List<Mechanism> mechanismList;
 
     /**
      * Creates Account object with given information.
-     * @param issuer String value of issuer
-     * @param displayIssuer String alternative value of the issuer
-     * @param accountName String value of accountName or username
+     *
+     * @param issuer             String value of issuer
+     * @param displayIssuer      String alternative value of the issuer
+     * @param accountName        String value of accountName or username
      * @param displayAccountName String alternative value of the accountName
-     * @param imageURL URL of account's logo image (optional)
-     * @param backgroundColor String HEX code of account's background color (optional)
-     * @param timeAdded Date and Time this Account was stored
+     * @param imageURL           URL of account's logo image (optional)
+     * @param backgroundColor    String HEX code of account's background color (optional)
+     * @param timeAdded          Date and Time this Account was stored
+     * @param policies           Policies used to enforce device security (optional)
+     * @param lockingPolicy      Indicates the policy locking the account (optional)
+     * @param lock               Indicates if the account is locked (optional)
      */
     public Account(String issuer, String displayIssuer, String accountName, String displayAccountName,
```

Comment:
Do we need to care about backward compatible? This will release as 4.0?

Reply:
No, this should be "protected", we need to fix it